docs/installing-northstar/using-northstar/playing-on-linux.md

```diff
@@ -155,15 +69,11 @@ Running the game on fullscreen through Linux might lead to a black screen preven
 
 For more info and proposed fixes, refer to [this issue thread on Github](https://github.com/R2Northstar/Northstar/issues/1)
 
-### LatencyFleX issues
-
-Some users have reported issues with enabling LatencyFleX. If you see `"Unable to load LatencyFleX library, LatencyFleX disabled."` in your logs, try adding `latencyflex_wine.dll` to your `bin/x64_retail` folder.
-
```

Comment:
Did we ever figure out the cause of this or did it just resolve itself over time (and us updating LatencyFlex dependencies and stuff)?

Reply:
Resolved with the latest LatencyFleX updates on our side

Reply:
(And also rendered impossible to occur under NorthstarProton)